test/mason/EXECENV

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env bash
+#!/USR/BIN/ENV BASH
```

Comment:
I'm not sure what happened here. Looks like things got capitalized?

I don't think we want to change this file.